test/js/main.js

```diff
@@ -38,31 +38,31 @@ const random = () => Math.random().toString(36).substring(7)
 const createDriver = ({name,localIdentifier}) => browser =>
   browser.remote ?
     new Builder()
-      .usingServer('http://hub-cloud.browserstack.com/wd/hub')
-      .withCapabilities({
-        ...bsCapabilitiesDefault,
-        ...browser,
-        name,
-        build: currentDate,
-        'browserstack.localIdentifier' : localIdentifier
-      })
+    .usingServer('http://hub-cloud.browserstack.com/wd/hub')
+    .withCapabilities({
+      ...bsCapabilitiesDefault,
+      ...browser,
+      name,
+      build: currentDate,
+      'browserstack.localIdentifier' : localIdentifier
+    })
     : new Builder().forBrowser(browser.browserName)
 
-
 const createBrowser = async (browser, testCase) => {
   const localIdentifier = random();
 
   const bsLocal = browser.remote ? await createBrowserStackLocal({
-		...browserstackLocalDefault,
+    ...browserstackLocalDefault,
     localIdentifier
   }) : null
 
-  const driver = await createDriver({name:testCase.file,localIdentifier})(browser)
-    .build();
+  const driver = await createDriver({name:testCase.file,localIdentifier})(browser).build();
```

Comment:
as much as it pains me to say this (Team Semicolons FTW!), for the sake of style consistency, this semicolon should be removed. 🙊

Reply:
You could also compact down how long this line is by indenting the object properties 

```
const driver = await createDriver({
    name:testCase.file,
    localIdentifier
})(browser).build()
```